### DISCUSSION TITLE
How to integrate with Django admin?

### DISCUSSION THREAD

**Discussion by napestershine** [agusmakmun/django-markdown-editor#244] [General]
Hi

I am trying to integrate it into my Django blog application. But in django admin I do not get any css and js. I did run the collectstatic. But it keep throwing error.
Any ideas?

**Comment by kaiserkonok** [comment]
Is this problem happening on production?

**Comment by agusmakmun** [comment]
Hello @napestershine let us know more details about your issue, so we can easily know the exact problem.

1. Explain what you have tried so far
2. Explain what is expected condition & what the current condition
3. Show any errors to us for debugging
4. Steps to reproduce the issue


Bonus references:
- https://docs.djangoproject.com/en/dev/howto/static-files/deployment/
- https://github.com/cookiecutter/cookiecutter-django
